src/module/postgame.js

```diff
@@ -0,0 +1,62 @@
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const submit = document.getElementById('submit');
+const ul = document.getElementById('ul');
+const refresh = document.getElementById('refresh');
+
+export const getGameId = localStorage.getItem('gameId');
+
+export const postData = (gameId) => {
+  const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`;
+  return fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name.value,
+      score: score.value,
+    }),
+  })
+    .then((response) => response.json())
+    .catch((error) => {
+      throw error;
+    });
```

Comment:
- [ ] Great job on fixing your `MIT` license as suggested by the previous reviewer. However, I noticed you didn't implement one of the required changes suggested by the previous reviewer which is also a project requirement that states that you should use  `async` and `await` features in JavaScript to consume the `API`as they are much cleaner and improves the readability of your code but instead, you're using `then`.  
 
 
 Kindly consider fixing this issue to fulfill the project requirement and also improve the readability of your code.

Reply:
I have changed where the code reviewer mentioned. and could not figure out other places can you be please be specific as I am burnt out and it is now 3:26 am in the morning. Regards

Reply:
@AbuTalha3 Sorry for the late reply, Kindly take a look at line `21`